sn_networking/src/record_store.rs

```diff
@@ -452,6 +435,32 @@ impl RecordStore for ClientRecordStore {
     fn remove_provider(&mut self, _key: &Key, _provider: &PeerId) {}
 }
 
+/// Cost calculator that increases cost nearing the maximum (2048).
+/// Table:
+///    1 =          0.000000010
```

Comment:
Are these values in full SNT, and so starting at 100 `Token`s?

Reply:
I thought that was 10 nanos, so 10^-8 tokens?